docker/cluster-test/cluster-test.Dockerfile

```diff
@@ -18,8 +18,8 @@ RUN rustup install $(cat rust-toolchain)
 COPY . /libra
 RUN docker/cluster-test/compile.sh
 FROM debian:buster
-RUN apt-get update && apt-get install -y openssh-client curl
-RUN curl -L "https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl" -o /usr/local/bin/kubectl && chmod +x /usr/local/bin/kubectl
+RUN apt-get update && apt-get install -y openssh-client curl wget
+RUN cd /usr/local/bin && wget "https://storage.googleapis.com/kubernetes-release/release/v1.17.0/bin/linux/amd64/kubectl" && chmod +x kubectl
```

Comment:
consider using -O --output-file for wget https://www.gnu.org/software/wget/manual/wget.html

Reply:
SG. Will udpate.